engine/table/src/main/java/io/deephaven/engine/util/PythonDeephavenSession.java

```diff
@@ -193,7 +194,9 @@ protected void evaluate(String command, String scriptName) {
 
     @Override
     public Map<String, Object> getVariables() {
-        return Collections.unmodifiableMap(scope.getEntriesMap());
+        final Map<String, Object> outMap = new LinkedHashMap<>();
+        scope.getEntriesMap().forEach((key, value) -> outMap.put(key, maybeUnwrap(value)));
+        return outMap;
```

Comment:
I'm wondering if `io.deephaven.engine.util.PythonScopeJpyImpl#convertValue` or `io.deephaven.engine.util.PythonScopeJpyImpl#convert` is the more appropriate place to do the unwrapping.